docusaurus/docs-internals/editor-controls/multivalue-controls/iconsPicker.md

```diff
@@ -269,11 +373,182 @@ When the helper object contains a position property with the value `"top-start"`
       title: "Item 2",
       value: "val2",
       icon: "nc-line"
-    },
+    }
   ],
   helper: {
     content: "Helper",
     position: "top-start"
   }
 }
-``` 
+```
+
+#### States examples
+
+Allows the control to work in normal and hover states.
+
+```js
+{
+  id: "icons",
+  type: "iconsPicker",
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    }
+  ],
+  states: ["normal", "hover"]
+}
+```
+
+Allows the control to work in normal, hover and active states.
+
+```js
+{
+  id: "icons",
+  type: "iconsPicker",
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    }
+  ],
+  states: ["normal", "hover", "active"]
+}
+```
+
+#### Default value examples
+
+In this example, the iconsPicker control that has the value `"val1"` by default.
+
+```js
+{
+  id: "icons",
+  type: "iconsPicker",
+  choices: [
+    {
+      title: "Item 1",
+      value: "val1",
+      icon: "nc-star"
+    },
+    {
+      title: "Item 2",
+      value: "val2",
+      icon: "nc-line"
+    }
+  ],
+  default: {
+    value: '["val1"]',
+    active: "val1"
+  }
+}
+```
+
+#### CSS examples
+
+Change the fontSize of the `.brz-text` element with CSS using custom values from a `iconsPicker` control.
+
+```js
+{
+  id: "icons",
+  type: "iconsPicker",
+  choices: [
+    {
+      title: "Small",
+      value: "small",
+      icon: "nc-small"
+    },
+    {
+      title: "Big",
+      value: "big",
+      icon: "nc-big"
+    }
+  ],
+  style: ({ value }) => {
+    return {
+      "{{WRAPPER}} .brz-text": {
+        fontSize: value.value === "small" ? 12 : 15;
+      }
+    }
+  }
+}
+```
+
+#### Usage in HTML example
+
+In the example below, we use the iconsPicker output value to determine the icon name in the button element.
+
+```tsx
+import { Brizy } from "@brizy/core";
+import React, { JSX } from "react";
+import { Icon } from "./Icon";
+
+interface Props {
+  icons: string;
+}
+
+const Button = (props: Props): JSX.Element => {
+  const { icons } = props;
```

Comment:
need more concrete example how to parse this
![image](https://github.com/EasyBrizy/Brizy-Local-Editor/assets/52622998/70a81f6c-a4e2-4834-acd1-7cec978a43c8)
and then use the values